pkg/config/settings/runtime_settings_test.go

```diff
@@ -105,7 +98,7 @@ func TestLogLevel(t *testing.T) {
 
 func TestProfiling(t *testing.T) {
 	cleanRuntimeSetting()
-	setupConf()
+	config.SetupConf()
```

Comment:
In the near future, I think we can replace this function 1:1 with `cfg := fxutil.Test[config.Component](t, config.MockModule)` 😄

Reply:
for sure. definitely a task for later 👍